testing/framework/test-framework.rst

```diff
@@ -165,6 +189,13 @@ into a directory to debug after a test has gone wrong.
 For a way around this, check out the ``PRESERVE`` environment variable.
 It can be seen in action in `How to convert old tests to use fixures`_ below.
 
+By the way, there's nothing magical about ``runtest.py``, the intent
```

Comment:
Not sure there's any benefit to encouraging anyone to submit PR with tests not using the current test infrastructure.
Just would lead to more maintenance burden.. (mainly for me)

Reply:
Hmmm. I meant if people didn't like our runner they could write their own, not that they'd send it to us. Maybe I was opening unwanted doors here... just wanted to make clear that unlike many test frameworks, ours rise only to the level of a single test and the separation was apparently, once, intentional.

Reply:
Yeah.. Let people know they can do this, and chance are someone who doesn't really understand it will do it and then burn a lot of support cycles.. 
If they're knowledgable enough to be able to do this correctly, they'll figure it out... others have.

I'd remove it lest we add another entry to pandora's box..